skgrid/tools/skgrid-mlib.cwl

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+baseCommand: ["/skgrid/mlib.sh"]
+hints:
+  DockerRequirement:
+    dockerPull: "skgrid"
```

Comment:
dockerPull vs dockerImageId https://www.commonwl.org/v1.0/CommandLineTool.html#DockerRequirement

Reply:
updated to dockerimageid to use human readable image name in commit [dd42380](https://github.com/ohsu-comp-bio/gdan-tmp-models/pull/7/commits/dd42380a2b7aec6ada1170eef28ade408a742ef9)